tools/prerequisites-check.sh

```diff
@@ -18,7 +18,7 @@ check_command_present "${DOCKER_CMD:-docker}"
 check_command_present shellcheck
 
 # After version 3.3.1, yq --version sends the string to STDERR instead of STDOUT
-YQ_VERSION="$(yq --version 2>&1 | ${SED} 's/^.* //g')"
+YQ_VERSION="$(yq --version 2>&1 | ${SED} -r 's/^.* v?//g')"
```

Comment:
Why was this change needed?

Reply:
i put it in a separated commit 8837187fadd682b28bcd878169fe25d2f198efa6 : 

```
 fix: yq version on mac os brew package is prefixed with v letter

yq version give `yq (https://github.com/mikefarah/yq/) version v4.30.5`
```

i can remote if you prefer. This is not really related to this feature.

Reply:
Looks like some change in the latest version of `yq`. `4.25.2` did not seems to have the `v`. Anyway, you can keep the change here - it works for me and it seems to work in the CI, so it should be fine.